packages/lib/src/components/buttons/NegotiateButtonComponent.wc.svelte

```diff
@@ -17,7 +17,7 @@
      */
     function exec(): void {
         if (type == "bbmri") {
```

Comment:
When we break the options anyway. We should also rename this to "NEGOTIATOR" and "PROJECTMANAGER". Please add https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change to your commit

Reply:
Honestly I'd feel more comfortable doing this when we have the typescript and JSON schema conversion in place and are running the options checker/catalogue checker again.

I'll start two discussion threads where we can collect breaking changes we want to make (one for options and one for catalogue).

Reply:
A suggestion: We could use PascalCase for enum-like strings everywhere. This would work well for tagged union discriminators because we can use the name of the typescript type (which is in PascalCase). And Rusts serde likes this as well when parsing enums.